<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Draughts Lite</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionSound"/>
    <addaction name="actionDraw"/>
    <addaction name="actionResign"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewGame">
   <property name="text">
    <string>创建新游戏</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>连接到游戏</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>音效</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>求和</string>
   </property>
   <property name="toolTip">
    <string>求和</string>
   </property>
  </action>
  <action name="actionResign">
   <property name="text">
    <string>认输</string>
   </property>
   <property name="toolTip">
    <string>认输 </string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
